### PR TITLE
Replace group when create space

### PR DESCRIPTION
**Pull request by darionyaphet**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### Checklist：
- [ ] Documentation affected （Please add the label if documentation needs to be modified.)
- [ ] Incompatible （If it is incompatible, please describe it and add corresponding label.）
- [ ] Need to cherry-pick （If need to cherry-pick to some branches, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes：
Please confirm whether to reflect in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Comment by SuperYoko** [comment]
use this to mark checkbox~
```
- [x] enhancement
```

File: src/graph/validator/AdminValidator.cpp [hunk 1]
```diff
@@ -20,8 +20,8 @@ Status CreateSpaceValidator::validateImpl() {
   ifNotExist_ = sentence->isIfNotExist();
   auto status = Status::OK();
   spaceDesc_.set_space_name(std::move(*(sentence->spaceName())));
-  if (sentence->groupName()) {
-    spaceDesc_.set_group_name(std::move(*(sentence->groupName())));
+  if (sentence->zoneNames()) {
+    spaceDesc_.set_zone_names(std::move(sentence->zoneNames()->zoneNames()));
```
Comment (SuperYoko) [inline src/graph/validator/AdminValidator.cpp:24]: why  `->zoneNames()->zoneNames()`? just curious.

File: src/meta/processors/zone/UpdateZoneProcessor.cpp [hunk 2]
```diff
@@ -119,41 +119,52 @@ void DropHostFromZoneProcessor::process(const cpp2::DropHostFromZoneReq& req) {
...
-    spaceIter->next();
+
+    if (code != nebula::cpp2::ErrorCode::SUCCEEDED) {
+      break;
+    } else {
+      spaceIter->next();
+    }
+  }
+
+  if (code != nebula::cpp2::ErrorCode::SUCCEEDED) {
```
Comment (SuperYoko) [inline src/meta/processors/zone/UpdateZoneProcessor.cpp:164]: maybe scope guard is okey to simplfy this logic?

File: src/graph/validator/AdminValidator.cpp (hunk 1 above)
Comment (darionyaphet) [reply src/graph/validator/AdminValidator.cpp:24]: It's just typed conversion and return the std::vector<std::string>

Comment (liwenhui-soul) [reply src/graph/validator/AdminValidator.cpp:24]: compile error by clang here. no need std::move

**Comment by darionyaphet** [comment]
rely on add host to fix describe space

File: src/meta/processors/zone/UpdateZoneProcessor.cpp (hunk 2 above)
Comment (darionyaphet) [reply src/meta/processors/zone/UpdateZoneProcessor.cpp:164]: after talk offline we think this is fine

**Comment by darionyaphet** [comment]
This PR have merged into [White list](https://github.com/vesoft-inc/nebula/pull/3343) so close it

**Pull request closed without merging**